lib/core/test/unit/Cardano/Wallet/Primitive/Types/TokenBundleSpec.hs

```diff
@@ -39,6 +41,46 @@ spec =
             , semigroupMonoidLaws
             ]
 
+    describe "Arithmetic" $ do
+        it "prop_difference_zero (x - 0 = x)" $
+            property prop_difference_zero
+        it "prop_difference_zero2 (0 - x = 0)" $
```

Comment:
Didn't want to break the naming convention too much, but I _did_ want to provide more readable expressions.

Reply:
Also, prop_difference_zero{1,2,3} could be unified into a single property, but I think this might be simpler/more readable in some sense.

Reply:
I think it's fine to have separate properties here. 👍🏻